docs/src/content/hardhat-network/docs/reference/index.md

```diff
@@ -289,6 +289,34 @@ const trace = await hre.network.provider.send("debug_traceTransaction", [
 
 #### `eth_call`
 
+The method allows you to call a contract's function without making a transaction on the blockchain. This is particularly useful for querying contract data and performing read-only operations. See the [GETH documentation](https://geth.ethereum.org/docs/interacting-with-geth/rpc/ns-eth) for more info.
```

Comment:
How the doc looks like in the website:

<img width="631" alt="image" src="https://github.com/NomicFoundation/hardhat/assets/18092467/01f9d645-4507-4a9c-b1e2-b3b4a061fb27">